draft-ietf-quic-http.md

```diff
@@ -777,18 +777,13 @@ Settings which are integers use the QUIC variable-length integer encoding.
 
 #### Defined SETTINGS Parameters {#settings-parameters}
 
-The following settings are defined in HTTP/QUIC:
-
-  SETTINGS_HEADER_TABLE_SIZE (0x1):
-  : An integer with a maximum value of 2^30 - 1.  The default value is 4,096
-    bytes.
+The following setting is defined in HTTP/QUIC:
 
   SETTINGS_MAX_HEADER_LIST_SIZE (0x6):
```

Comment:
Does anything in this document explain what SETTINGS_MAX_HEADER_LIST_SIZE does?

Reply:
Good point.  RFC7540 doesn't have much, but it has more than HQ does currently.